src/test/java/com/connexta/search/index/BodyContentExtractorTest.java

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (c) 2019 Connexta, LLC
+ *
+ * Released under the GNU Lesser General Public License version 3; see
+ * https://www.gnu.org/licenses/lgpl-3.0.html
+ */
+package com.connexta.search.index;
+
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.containsString;
+import static org.hamcrest.Matchers.emptyString;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.io.IOException;
+import java.io.InputStream;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+public class BodyContentExtractorTest {
+
+  private static final String SAMPLE_PDF = "sample.pdf";
+
+  private static final String SAMPLE_DOC = "file-sample_100kB.doc";
+
+  private static final String SAMPLE_DOCX = "file-sample_100kB.docx";
+
+  private static final String SAMPLE_PPT = "file_example_PPT_250kB.ppt";
+
+  private static final String SAMPLE_PPTX = "sample-pptx.pptx";
+
+  private static final String SAMPLE_XLS = "file_example_XLS_10.xls";
+
+  private static final String SAMPLE_XLSX = "file_example_XLSX_10.xlsx";
+
+  private static final String SAMPLE_JPG = "file_example_JPG_100kB.jpg";
+
+  private static final String SAMPLE_TXT = "sample1.txt";
+
+  @Test
+  @DisplayName("Text Extraction from PDF")
+  public void testExtractTextPdf() throws Exception {
+    final String EXPECTED_WORD = "demonstration";
+    final BodyContentExtractor bodyContentExtractor = new BodyContentExtractor();
```

Comment:
✏️ If testMaxContentLength is removed (see other comment), bodyContentExtractor can be made a class field instead of constructing it in every test method.